framework/source/class/qx/bom/client/Event.js

```diff
@@ -212,7 +212,31 @@ qx.Bootstrap.define("qx.bom.client.Event",
       catch(ex) {};
       
       return (hasAuxclick ? true : false);
-    }
+    },
+
+    /**
+     * Checks whether the browser supports passive event handlers.
+     */
+    getPassive: function () {
+      var passiveSupported = false;
+      try {
+        var options = {
+            get passive() {
```

Comment:
I guess the generator will hate this ...

Reply:
@johnspackman Could we create an environment variable or does one exist that is set only for the compiler, and then `qx.core.Environment.select` the code so that the generator will ignore it? Or doesn't this work this way?

Reply:
@lucapivato For qooxdoo v6, we still need to be compatible with the python generator, which does not understand modern javascript. Have you tried compiling a test app with the generator yet? If it chokes on the getter syntax, maybe you'd have to use eval() to do the test.

Reply:
I will change it to use eval(""). I don't use the generator for the qooxdoo library in wisej, I have a single bundled minified visual studio project that builds qx.js.